var/spack/repos/builtin/packages/python/package.py

```diff
@@ -455,6 +453,13 @@ def flag_handler(self, name, flags):
             if name == 'cflags':
                 flags.append('-fwrapv')
 
+        # Fix for following issues for python with aocc%3.2.0:
+        # https://github.com/spack/spack/issues/29115
+        # https://github.com/spack/spack/pull/28708
+        if self.spec.satisfies('%aocc@3.2.0', strict=True):
+            if name == 'cflags':
+                flags.extend(['-mllvm','-disable-indvar-simplify=true'])
```

Comment:
```suggestion
                flags.extend(['-mllvm', '-disable-indvar-simplify=true'])
```
This should fix the style tests